commit 71251dbe4c2ff70b4abdfdba29ba41d85fccb417 (refs/changes/07/3407/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-16T15:21:38-08:00 (1 year, 8 months ago)
    
    TRITON-139 Triton's config-agents should poll at 120s interval as a stopgap
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index 9cd96a3..bda632a 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit 9cd96a35d84c223485e14f02e7a4ebca2088ca51
+Subproject commit bda632a69ff44114f9faf689a5cd4af1fe578d66
